content/qemu/index.md

```diff
@@ -3,40 +3,97 @@ title = 'QEMU'
 icon = 'clanbomber'
 +++
 
-<a href="https://www.qemu.org/">QEMU</a> is an emulator for
-various CPUs. It works on Linux, Windows, FreeBSD and Mac OS X. It's possible
-to run grml with QEMU. Notice that it's much slower than running it native
-due to the emulation mode.
+[QEMU](https://www.qemu.org/) is an emulator for various CPUs.
+It works on Linux, Windows, FreeBSD and Mac OS X.
+It's possible to run grml with QEMU.
```

Comment:
nitpick: Grml (instead of grml).

(I did not comment every instance).

Reply:
fixed